### PR TITLE
finalize survey url

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

File: src/web/lib/components/AppHeader/index.js [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 import iconFeedback from "./feedback.svg";
 import "./index.scss";
 
-export const AppHeader = ({ hasExtension, appVersion = null }) => {
-  let survey = `${surveyUrl}?ref=app`;
-  if (appVersion) {
-    survey += `${survey}&ver=${appVersion}`;
-  }
+export const AppHeader = ({ hasExtension }) => {
+  const survey = `${surveyUrl}?ref=app?ver=${version}`;
```
Comment (lmorchard) [inline src/web/lib/components/AppHeader/index.js]: The second `?` here should be a `&` - otherwise you we get an URL with `?ref=app?ver=0.0.35`

**Comment by lmorchard** [comment]
Realized I could make the tweak myself, so did that and will merge after tests pass

**Comment by johngruen** [comment]
@lmorchard sorry i missed your comment...i knocked off a bit early on Friday. Should be ready to go now 👍

**Comment by johngruen** [comment]
Merging since i updated based on comments

**Pull request merged**